home/tests/test_explore_page.py

```diff
@@ -39,3 +39,11 @@ def test_explore_page_response(self, explore_page_response):
         def test_new_question_btn_in_explore_page(self, explore_page_response):
             char_content = explore_page_response.content.decode(explore_page_response.charset)
             assert '<a href="%s"' % reverse("new_question") in char_content
+
+        @pytest.mark.parametrize("tag_name", [
+            '#Pitagoras',
+            '#Bagrut_Exam'
+        ])
+        @pytest.mark.django_db
+        def test_explore_page_tags_view(self, explore_page_response, tag_name):
+            assert tag_name in str(explore_page_response.content)
```

Comment:
It looks like that you assume that you have at least one question with the tags "Pitagoras" and "Bagrut_Exam".
If it will be changed, for example by the "edit" option of the user or other option, that test will fail. 
I think these test should be like that :
1. add a question with tags that don't exist (check it before).
2. make this test with the new tags that you entered.

What do you think?

Reply:
I don't think we have an "edit" option, so I think it ok to leave this 2 tags.
I agree with aviv on adding maybe 2 new tags.

Reply:
I do not think it matters that we didn't implement that feature, equally the admin can delete the question for some reason, and then these tags will not exist.
The point is that I do not think we should rely on data that we actually have in the app when performing the tests, and therefore she should add a question with tags that do not exist in the app.

Reply:
Most of our test rely on data we have in the app.

Reply:
@avivz450 
I test it this way because we don't allow the user to delete / edit questions (and not plan to do so ) therefore those tags should appear In the explore page. 
 Moreover, I agree with Rebecca that most of our tests rely on data we have in the app and therefore it seems that we can rely on them on the tests.
 @ifireball   I would be glad to hear your opinion regarding this issue.

Reply:
Not sure I understand @avivz450 point - the test data is there in order to be used in tests.

Reply:
This is irrelevant, I thought you were relying on the test data that was deleted after deleting db.sqlite3 and rebooting the virtual machine (using `vagrant halt`, `vagrant destroy`, and `vagrant up`).
Anyway, I realized this is not what I thought. I think that this test is great :)